pkgs/development/python-modules/subliminal/default.nix

```diff
@@ -16,7 +16,7 @@
 , appdirs
 , rarfile
 , pytz
-, futures
+, futures ? null
```

Comment:
what does this?

Reply:
futures is only available in python2 IIRC.